docs/controls/InAppNotification.md

```diff
@@ -88,9 +88,50 @@ To hide it, simply set the property to `ShowDismissButton="False"`.
 
 ## Events
 
+### Opening
+
+This event is raised when the system or your user started to open of the notification.
```

Comment:
The user would never be directly triggering the notification.  Maybe:

    This event is raised just before the notification starts to open.